content/03.results.md

```diff
@@ -71,7 +72,7 @@ While DNA k-mers can provide resolution at the genus level, log-transformed cont
 Average containment estimated from proteome sequence is very similar to anchor containment estimated from 6-frame translation of genome sequence, suggesting that either value can be used for this type of comparison.
 We obtained similar results when comparing all available k-mers, suggesting that these results are not affected by FracMinHash scaling (_Supplemental Figure XX_).
 
-![**Protein k-mers are shared at higher taxonomic ranks** Default scaled values 1000, 200](images/gtdb-rs202.evolpaths.alphacompare.avgContain.logscale.ridgeplot.png){#fig:evolpaths-lca-containment height=4in}
+![**Protein k-mers are shared at higher taxonomic ranks** Default scaled values 1000, 200. CTB: provide log scale on x axis.](images/gtdb-rs202.evolpaths.alphacompare.avgContain.logscale.ridgeplot.png){#fig:evolpaths-lca-containment height=4in}
```

Comment:
The x is displayed in log - how were you thinking of modifying?

Reply:
it needs to be indicated on the axis that it's log, I think? It's obvious when you look at the numbers themselves.